packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -42,8 +43,8 @@ export type MultichainAccountServiceOptions = {
   messenger: MultichainAccountServiceMessenger;
   providers?: Bip44AccountProvider[];
   providerConfigs?: {
-    [EvmAccountProvider.NAME]?: EvmAccountProviderConfig;
-    [SolAccountProvider.NAME]?: SolAccountProviderConfig;
+    [EVM_ACCOUNT_PROVIDER_NAME]?: EvmAccountProviderConfig;
+    [SOL_ACCOUNT_PROVIDER_NAME]?: SolAccountProviderConfig;
```

Comment:
Not sure why this is popping now, but I got building errors when using this instead of the actual constant (which should literally be the same...):

```console
packages/multichain-account-service/src/MultichainAccountService.ts:45:5 - error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
```